opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -1379,15 +1379,32 @@ export const resolveUserById = async (context, id) => {
   return buildCompleteUser(context, client);
 };
 
-const resolveUserByToken = async (context, tokenValue) => {
+export const resolveUserByToken = async (context, tokenValue) => {
   const client = await elLoadBy(context, SYSTEM_USER, 'api_token', tokenValue, ENTITY_TYPE_USER);
   return buildCompleteUser(context, client);
 };
 
 export const userRenewToken = async (context, user, userId) => {
+  const userData = await storeLoadById(context, user, userId, ENTITY_TYPE_USER);
+  if (!userData) {
+    throw FunctionalError(`Cannot renew token, ${userId} user cannot be found.`);
+  }
```

Comment:
I think we need to protect this API when it's an admin orga that calls it

Reply:
ah yes thanks

Reply:
protected at graphql level with `auth`